addon/components/ember-youtube.js

```diff
@@ -137,21 +145,25 @@ export default Component.extend({
 				// If already loaded, make sure not to load the script again.
 				resolve(window.YT);
 			} else {
-				$.getScript('https://www.youtube.com/iframe_api');
+				let ytScript = document.createElement("script");
```

Comment:
since `ember-ajax` and `jquery` have been removed, `$.getScript` has been replaced with a native solution borrowed from jQuery.

Under the hood, all jQuery `getScript` was doing was creating a `<script>` tag and appending it to the page, thus evaluating the script on page. Here I just do that in 3 lines of code.